### DISCUSSION TITLE
No Dashboard tab in home page

### DISCUSSION THREAD

**Discussion by zlsfleet** [teslamate-org/teslamate#2222] [Q&A] [answered]
This is what my homepage looks like: 
![dc5eb81321874c801e1de627803f935](https://user-images.githubusercontent.com/1870876/145919604-c54094a9-6ee9-485c-8099-01b3e072e70b.png)

There is no Dashboard tab which is supposed to show up on the top right. Teslamate was installed on a clean Ubuntu 20.04 by docker script successfully. Then I checked the log and the only errors I get are "teslamate_1  | 2021-12-14 01:55:41.362 [error] GET https://nominatim.openstreetmap.org/reverse -> error: "timeout" (5001.496 ms) ".  But I suppose this is not related?

The grafana home page be like:
![a4f1fe69a255c6e0523d84fbc9a05a5](https://user-images.githubusercontent.com/1870876/145920006-e3c0bdf6-7030-4309-b7d4-9fd40e6fa9ca.png)

I really don't know what to do now. Please help. Thanks.

**Comment by metaben** [comment]
The code has
`<%= if grafana_url = @conn.assigns.settings.grafana_url do %>`

Maybe you changed the grafana_url value?

**Comment by adriankumpf** [comment, marked as answer]
You can either set the Grafana URL manually on the Settings page: `URLs -> Dashboards` or click on any address in one of the Grafana dashboards. Once you're redirected to the geofence page, the Grafana URL will be set automatically for you.

**Reply by zlsfleet** [reply]
Thanks, it works.
